packages/core/src/App/Components/Layout/Header/Components/ToggleMenu/menu-title.tsx

```diff
@@ -23,13 +24,12 @@ const MenuTitle = observer(() => {
                 >
                     {!is_mobile_language_menu_open && (
                         <React.Fragment>
-                            <Icon
-                                icon={`IcFlag${current_language.replace('_', '-')}`}
-                                data_testid='dt_icon'
-                                className='ic-settings-language__icon'
-                                size={22}
-                            />
-                            <Text weight='bold' size='xxs'>
+                            {TranslationFlag[current_language] ? (
+                                TranslationFlag[current_language]('xs')
+                            ) : (
+                                <Icon icon={`IcFlag${current_language}`} data_testid='dt_icon' size={18} />
```

Comment:
Please add a TODO for the KO language here as well. Or is there other reason for having this fallback?